docs/sdk/server-side-sdks/java-local/java-local-install.md

```diff
@@ -11,20 +11,29 @@ sidebar_custom_props: { icon: screwdriver-wrench }
 
 ## Requirements
 
-This version of the DevCycle SDK works with Java 8 and above.
+This version of the DevCycle SDK works with Java 11 and above.
 
-Using the Java SDK library requires [Maven](https://maven.apache.org/) or [Gradle](https://gradle.org/) >= 5.6.4 to be installed.
+Using the Java SDK library requires [Maven](https://maven.apache.org/) or [Gradle](https://gradle.org/) >= 7.6+ to be installed.
 
 :::info
 
-An x86_64 JDK is required for Local Bucketing with the DevCycle Java SDK. Currently Supported Platforms are:
+An x86_64 or aarch64 JDK is required for Local Bucketing with the DevCycle Java SDK.
+
+Currently Supported Platforms are:
 
 | OS          | Arch   |
 | ----------- | ------ |
 | Linux (ELF) | x86_64 |
 | Mac OS      | x86_64 |
+| Mac OS      | aarch64 |
```

Comment:
Do we not support linux aarch64 too now?

Reply:
yup, forgot that one.  fixed.